lib/lib_display/UDisplay/uDisplay.cpp

```diff
@@ -2229,42 +2229,50 @@ void uDisplay::pushColors(uint16_t *data, uint16_t len, boolean not_swapped) {
     // check that bytes count matches the size of area, and remove from inner loop
     if ((seta_yp2 - seta_yp1) * (seta_xp2 - seta_xp2) > len) { return; }
 
```

Comment:
I would be more comfortable with `uint32_t` if we ever have bigger displays. Actually forcing `uint16_t` most certainly makes the code bigger

Reply:
Never mind, it's 16 bits all over the place

Reply:
this indeed means that we may not push more than 65k pixels and was inherited from very old drivers
the :B parameter in the display descriptor thus has a limit.
we may change this to 32 bit.